From 708b032f7880309b9e6746ce76c1309005943eb6 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler <raster@openmoko.org>
Date: Tue, 14 Apr 2009 20:27:11 +1000
Subject: [PATCH] glibc: write ld.so.cache to usual /etc location so it can be cached and speed
 up boot (so it doesnt need to re-generate every boot!)

---
 recipes/glibc/glibc_2.6.1.bb |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/recipes/glibc/glibc_2.6.1.bb b/recipes/glibc/glibc_2.6.1.bb
index ab343ea..c1a9598 100644
--- a/recipes/glibc/glibc_2.6.1.bb
+++ b/recipes/glibc/glibc_2.6.1.bb
@@ -41,7 +41,6 @@ SRC_URI = "\
   file://arm-longlong.patch;patch=1 \
   file://fhs-linux-paths.patch;patch=1 \
   file://dl-cache-libcmp.patch;patch=1 \
-  file://ldsocache-varrun.patch;patch=1 \
   file://nptl-crosscompile.patch;patch=1 \
   file://glibc-2.5-local-dynamic-resolvconf.patch;patch=1;pnum=0 \
   file://glibc-check_pf.patch;patch=1;pnum=0 \
@@ -61,6 +60,8 @@ SRC_URI = "\
   file://march-i686.patch;patch=1;pnum=0 \
 "
 
+#  file://ldsocache-varrun.patch;patch=1 \
+
 # Build fails on sh3 and sh4 without additional patches
 SRC_URI_append_sh3 = " file://no-z-defs.patch;patch=1"
 SRC_URI_append_sh4 = " file://no-z-defs.patch;patch=1"
-- 
1.5.6.3

